dashboard/config/foorm/forms/surveys/pd/summer_workshop_pre_survey.0.json

```diff
@@ -1,1200 +1,663 @@
+    },
+    {
+     "type": "rating",
+     "name": "material_ratings",
+     "indent": 12,
+     "title": "Instructional focus. 1. more material,some students struggle --- 5. less material, leave fewer behind",
+     "titleLocation": "hidden",
+     "minRateDescription": "Strongly aligned with A",
+     "maxRateDescription": "Strongly aligned with B"
+    }
+   ],
+   "title": "Teaching Philosophy"
+  },
+  {
+   "name": "wrap_up_submit",
+   "elements": [
+    {
+     "type": "radiogroup",
+     "name": "highest_level_cs_education",
+     "title": "Which most accurately describes the highest level of formal computer science education you have completed? ",
+     "hasOther": true,
+     "otherPlaceHolder": "Other CS Education",
+     "choices": [
+      {
+       "value": "little_no",
+       "text": "At this time, I have little to no formal education in computer science."
+      },
+      {
+       "value": "online_courses",
+       "text": "I have taken one or more courses online for credit."
+      },
+      {
+       "value": "college_courses",
+       "text": "I took one or more courses in college."
+      },
+      {
+       "value": "earning_degree",
+       "text": "I am currently in the process of earning a computer science degree."
+      },
+      {
+       "value": "bachelors_degree",
+       "text": "I have a Bachelor's degree in computer science."
+      },
+      {
+       "value": "masters_degree",
+       "text": "I have a Master's degree in computer science."
+      },
+      {
+       "value": "phd",
+       "text": "I have a PhD in computer science."
+      }
+     ],
+     "otherText": "Other CS Education"
+    },
+    {
+     "type": "radiogroup",
+     "name": "teaching_plan",
+     "title": "Which of the following most closely applies to you about teaching {workshop_course} next school year: ",
+     "choices": [
+      {
+       "value": "assigned",
+       "text": "I was assigned to teach the course."
+      },
+      {
+       "value": "chose",
+       "text": "I chose to teach the course."
+      },
+      {
+       "value": "not_planning",
+       "text": "I am not currently planning to teach the course next year."
+      }
+     ]
+    },
+    {
+     "type": "text",
+     "name": "birth_year",
+     "title": "What is your year of birth? ",
+     "description": "Please enter a whole number e.g. 1975. ",
+     "inputType": "number",
+     "placeHolder": "enter a number"
```

Comment:
don't need to add this now, but you can put a min/max on this field so that when the user clicks the arrows on the side it won't start at 1 for birth year, and they can't enter a 5 digit year, for example add:
```
"min": "1920",
"max": "2020"
```

Reply:
Done